cli-plugins/apps/debug-workload.hbs.md

```diff
@@ -50,14 +50,14 @@ There are known errors that will make the workload enter in an error or unknown
 			- Fix the path in the `—source-image` flag value to point to a valid repo path.
 
 - *WorkloadLabelsMissing* / *SupplyChainNotFound*
-	- *Message*: No supply chain found where full selector is satisfied by labels: map[app.kubernetes.io/part-of:spring-petclinic]
+	- *Message*: No supply chain found where full selector is satisfied by `labels: map[app.kubernetes.io/part-of:spring-petclinic]`
 		- *Cause*: The labels and attributes in the workload object did not fully satisfy any installed supply chain on the cluster.
 		- *Resolution*: Use the `tanzu apps csc list` and `tanzu apps csc get <supply-chain>` commands to see the selector criterias for the supply chains. You can apply the missing labels to a workload by using `tanzu apps workload apply`
```

Comment:
- *Resolution*: Use the `tanzu apps cluster-supply-chain list` (alias `csc`) and `tanzu apps csc get <supply-chain-name>` commands to see the workload selection criteria for the supply chain(s) available on the cluster. You can apply any missing labels to a workload by using `tanzu apps workload apply --label required-label-name=required-label-value`

Reply:
updated